src/nvim/po/de.po

```diff
@@ -19,132 +19,375 @@ msgstr ""
+#, fuzzy
+msgid "Index out of bounds"
+msgstr "Index out of bounds"
+
+#: ../api/buffer.c:186
+msgid "Line index is too high"
+msgstr "Zeilenindex ist zu gro�"
+
+#: ../api/buffer.c:289
+msgid "Argument \"start\" is higher than \"end\""
+msgstr "Argument \"start\" is gr��er als \"end\""
+
+#: ../api/buffer.c:304
+msgid "All items in the replacement array must be strings"
+msgstr "Alle List-Eintr�ge in Ersatzarray m�ssen Zeichenfolgen sein"
+
+#: ../api/buffer.c:315
+msgid "string cannot contain newlines"
+msgstr "Zeichenfolge kann keine Zeilenumbr�che enthalten"
+
+#: ../api/buffer.c:328 ../ops.c:2729
+msgid "Failed to save undo information"
+msgstr "Speichern der Wiederherstellunginfo gescheitert"
+
+#: ../api/buffer.c:338
+msgid "Failed to delete line"
+msgstr "L�schen der Zeile gescheitert"
+
+#: ../api/buffer.c:355 ../api/buffer.c:373
+msgid "Index value is too high"
+msgstr "Indexwert ist zu gro�"
+
+#: ../api/buffer.c:360
+msgid "Failed to replace line"
+msgstr "Ersetzen der Zeile gescheitert"
+
+#: ../api/buffer.c:378
+msgid "Failed to insert line"
+msgstr "Einf�gung der Zeile gescheitert"
+
+#: ../api/buffer.c:616
+msgid "Failed to rename buffer"
+msgstr "Umbenennung der Lesepuffer gescheitert"
+
+#: ../api/buffer.c:665
+msgid "Mark name must be a single character"
+msgstr "Name einer Markierung muss ein Einzelzeichen sein"
+
+#: ../api/buffer.c:683
+msgid "Invalid mark name"
+msgstr "Ung�ltiger Name einer Markierung"
+
+#: ../api/buffer.c:737 ../api/buffer.c:777
+msgid "Line number outside range"
+msgstr "Zeilennummer au�erhalb des zul�ssigen Bereichs"
+
+#: ../api/buffer.c:741 ../api/window.c:80
+msgid "Column value outside range"
+msgstr "Spaltennummer au�erhalb des zul�ssigen Bereichs"
+
+#: ../api/private/helpers.c:60
+msgid "Keyboard interrupt"
+msgstr "Tastatur-Unterbrechung"
+
+#: ../api/private/helpers.c:93
+msgid "Key not found"
+msgstr "Pfad nicht gefunden "
+
+#: ../api/private/helpers.c:118
+#, fuzzy
+msgid "Dictionary is locked"
+msgstr "Dictionary gesperrt"
+
+#: ../api/private/helpers.c:123
+msgid "Empty variable names aren't allowed"
+msgstr "Variablennamen k�nnen nicht leer sein"
+
+#: ../api/private/helpers.c:128
+msgid "Key length is too high"
+msgstr "Die L�nge des Pfads ist zu gro�!"
+
+#: ../api/private/helpers.c:136
+#, c-format
+msgid "Key is read-only: %s"
+msgstr ""
+
+#: ../api/private/helpers.c:139
+#, c-format
+msgid "Key is fixed: %s"
+msgstr ""
+
+#: ../api/private/helpers.c:142
+#, fuzzy, c-format
+msgid "Key is locked: %s"
+msgstr "E741: Wert ist gesperrt: %s"
+
+#. Doesn't exist, fail
+#: ../api/private/helpers.c:151
+#, fuzzy, c-format
+msgid "Key \"%s\" doesn't exist"
+msgstr "Die Datei \"%s\" existiert nicht"
+
+#: ../api/private/helpers.c:205 ../api/private/helpers.c:260
+#, fuzzy
+msgid "Empty option name"
+msgstr "E792: Leerer Men�name"
+
+#: ../api/private/helpers.c:218 ../api/private/helpers.c:269
+#, fuzzy, c-format
+msgid "Invalid option name \"%s\""
+msgstr "E755: Ung�ltige Region in %s"
```

Comment:
@mschilli87 the `# fuzzy` above means it was translated automatically by the script. I still gotta sort those out.